### DISCUSSION TITLE
Maybe a "bug" in your pricing page

### DISCUSSION THREAD

**Discussion by bmcgonag** [rustdesk/rustdesk#6789] [General]
Not sure if this is intentional, but you have the personal plan as $ 9.90 / month billed annually for 2 users and 100 managed devices.  

I can, however, go to the customize pricing option on that page and enter 5 users, and 500 devices, and get pricing of $ 4.97 / month billed annually.  

Haven't tried to check out, so maybe it adjusts during payment, but it's odd, and might lead to lost revenue for you.  Just an FYI.
![Screenshot from 2024-01-03 14-54-40](https://github.com/rustdesk/rustdesk/assets/7346620/6be293fa-c5f7-44e0-a5c2-1c609a98e354)
![Screenshot from 2024-01-03 14-58-38](https://github.com/rustdesk/rustdesk/assets/7346620/6e0f4ce9-105a-4721-a1e2-8eca66273add)

**Comment by moreamazingnick** [comment]
or we take that plan :) =>
<img width="235" alt="Screenshot 2024-01-05 091817" src="https://github.com/rustdesk/rustdesk/assets/29682206/32a1c074-cebd-45fb-94b8-d600c01478d9">

**Reply by moreamazingnick** [reply]
too bad ...
<img width="381" alt="Screenshot 2024-01-05 092026" src="https://github.com/rustdesk/rustdesk/assets/29682206/9e71a192-31c8-48ea-aa5f-6d0ca3d923a6">

**Comment by moreamazingnick** [comment]
it is fixed...
